### ISSUE TITLE
Run pyright with poetry

### ISSUE THREAD

**Issue by sylvainmouquet** [microsoft/pyright#3424]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**
When i run pyright cli there is no errors. When i run it by poetry i have one error

**Screenshots or Code**
If applicable, add screenshots or the text of the code (surrounded by triple back ticks) to help explain your problem.
```python
sylvain$ pyright --version
pyright 1.1.244
sylvain$ poetry run pyright --version
pyright 1.1.244
sylvain$ poetry run pyright app/main_standalone.py 
Loading configuration file at /<path>/pyrightconfig.json
stubPath /<path>/typings is not a valid directory.
Searching for source files
Found 1 source file
/<path>/app/main_standalone.py
  /<path>/app/main_standalone.py:43:43 - error: Argument missing for parameter "jwt_accesstoken" (reportGeneralTypeIssues)
1 error, 0 warnings, 0 informations 
Completed in 1.759sec
sylvain$ pyright app/main_standalone.py 
Loading configuration file at <path>/pyrightconfig.json
stubPath <path>/typings is not a valid directory.
Searching for source files
Found 1 source file
0 errors, 0 warnings, 0 informations 
Completed in 1.665sec

```

**Comment by sylvainmouquet** [comment]
`abstract_member: AbstractMember = AbstractMember(ldap=payload.payload["ldap"])
`
````
from pydantic import BaseModel

class AbstractMember(BaseModel):
    ldap: Optional[str]
    jwt_accesstoken: Optional[str]

    class Config:
        arbitrary_types_allowed = True


```

**Comment by sylvainmouquet** [comment]
there is no errors when i add a value to the optional :

```
class AbstractMember(BaseModel):
    ldap: Optional[str] = None
    jwt_accesstoken: Optional[str] = None

    class Config:
        arbitrary_types_allowed = True
```
